m/configDataPreprocessingSeaexplorer.m

```diff
@@ -0,0 +1,129 @@
+% $$$   preprocessing_options.position_list(1).conversion = @nmea2deg;
+  
+  
+  %% CTD (each serie is a different configuration of glider sensors)
+  %  **All variables from a serie must be found to go forward**
+  preprocessing_options.ctd_list(1).conductivity = 'GPCTD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(1).temperature  = 'GPCTD_TEMPERATURE';
+  preprocessing_options.ctd_list(1).pressure     = 'GPCTD_PRESSURE';
+  preprocessing_options.ctd_list(1).oxygen       = 'GPCTD_DOF';
+
+  preprocessing_options.ctd_list(2).conductivity = 'SBD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(2).temperature  = 'SBD_TEMPERATURE';
+  preprocessing_options.ctd_list(2).pressure     = 'SBD_PRESSURE';
+  preprocessing_options.ctd_list(2).oxygen       = 'SBD_DOF';
+
+  preprocessing_options.ctd_list(3).conductivity = 'GPCTD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(3).temperature  = 'GPCTD_TEMPERATURE';
+  preprocessing_options.ctd_list(3).pressure     = 'GPCTD_PRESSURE';
+
+  %% Optics
+  preprocessing_options.optics_list(1).chlorophyll = 'FLBBCD_CHL_SCALED';
+  preprocessing_options.optics_list(1).cdom        = 'FLBBCD_CDOM_SCALED';
+  preprocessing_options.optics_list(1).backscatter_700 = 'FLBBCD_BB_700_SCALED';
+
+  preprocessing_options.optics_list(2).chlorophyll = 'TRI_CHL_SCALED';
+  preprocessing_options.optics_list(2).cdom        = 'TRI_CDOM_SCALED';
+  preprocessing_options.optics_list(2).backscatter_700 = 'TRI_BB_700_SCALED';
+
+  preprocessing_options.optics_list(3).chlorophyll = 'FLNTU_CHL_SCALED';
+  preprocessing_options.optics_list(3).turbidity = 'FLNTU_NTU_SCALED';  
+  
+  
+  %% MFL
+  preprocessing_options.mfl_list(1).mfl_v1 = 'MFL_V1';
+  preprocessing_options.mfl_list(1).mfl_v2 = 'MFL_V2';
+  preprocessing_options.mfl_list(1).mfl_v3 = 'MFL_V3';
+  preprocessing_options.mfl_list(1).mfl_v4 = 'MFL_V4';
+  preprocessing_options.mfl_list(1).mfl_te = 'MFL_TMPE';
+  preprocessing_options.mfl_list(1).mfl_td = 'MFL_TMPD';  
+
+  preprocessing_options.mfl_list(2).mfl_v1 = 'UV1_V1';
+  preprocessing_options.mfl_list(2).mfl_v2 = 'UV1_V2';
+  preprocessing_options.mfl_list(2).mfl_v3 = 'UV1_V3';
+  preprocessing_options.mfl_list(2).mfl_v4 = 'UV1_V4';  
+  preprocessing_options.mfl_list(2).mfl_te = 'UV1_TMP1';
+  preprocessing_options.mfl_list(2).mfl_td = 'UV1_TMP2';  
+
+  preprocessing_options.mfl_list(3).mfl_v1 = 'UV1_PHE';
+  preprocessing_options.mfl_list(3).mfl_v2 = 'UV1_TRY';
+  preprocessing_options.mfl_list(3).mfl_v3 = 'UV1_LD1';
+  preprocessing_options.mfl_list(3).mfl_v4 = 'UV1_LD2';
+  preprocessing_options.mfl_list(3).mfl_te = 'UV1_TMP1';
+  preprocessing_options.mfl_list(3).mfl_td = 'UV1_TMP2';  
+
+  %% METS
+  preprocessing_options.mets_list(1).methane_volt = 'METS_METHANE_VOLT';
+  preprocessing_options.mets_list(1).methane_scaled = 'METS_METHANE_SCALED';
+  preprocessing_options.mets_list(1).mets_temp_volt = 'METS_TEMP_VOLT';
+  preprocessing_options.mets_list(1).mets_temp_scaled = 'METS_TEMP_SCALED';
```

Comment:
No worries, the manual of the glider is being written. I am also writing numerous email to the manufacturer to have more info about variables.

METS It is a methane sensor. Together with the MFL sensor (fluorometer that measures dissolved organic matter and other hydrocarbons), they are unique to SeaExplorer. They are routinely deployed on these gliders so I thought they should have their own list... But it is your toolbox, you should decide where to put them.